normandy/base/api/mixins.py

```diff
@@ -0,0 +1,14 @@
+from django.conf import settings
+from django.views.decorators.cache import cache_control
+
+
+class CachingMixin:
```

Comment:
I think we decided not to really make a decision. I'd be happy to find a
lint either way for it.

On Thu, Jun 23, 2016 at 12:55:54PM -0700, Michael Kelly wrote:
> > @@ -0,0 +1,14 @@
> > +from django.conf import settings
> > +from django.views.decorators.cache import cache_control
> > +
> > +
> > +class CachingMixin:
> 
> I can't remember what we decided RE: subclassing `object` or not. A lot of our test classes do, for example, as do some middlewares and other classes around the codebase I found do.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/mozilla/normandy/pull/172/files/38110cadcd17ed7618ed2ad2784169ac7ae36258#r68303498